### PR TITLE
removes vlad

### PR DESCRIPTION
**Pull request by ArchieBeepBoop**
## About The Pull Request

pr speedrun any%

### CODE REVIEW

**Review by Dahlular — APPROVED** [review]
yea lol

**Comment by SmeagleDeagle** [comment]
You did it, you killed my boy.

File: code/modules/mob/dead/new_player/login.dm [hunk 1]
```diff
@@ -13,6 +13,9 @@
 	if(motd)
 		to_chat(src, "<div class=\"motd\">[motd]</div>", handle_whitespace=FALSE)
 
+	if (mind.key == "Vladoricious")
```
Comment (VladinXXV) [inline code/modules/mob/dead/new_player/login.dm:16]: isnt this supposed to say HackerT.Dog

**Review by VladinXXV — CHANGES_REQUESTED** [review]
can we not though

**Pull request closed without merging**